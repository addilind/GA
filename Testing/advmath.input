5+2*3;
(5+2)*3;
5-3;
5.0-2.5;
5.0/2.0;