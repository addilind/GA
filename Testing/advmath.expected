11\n21\n2\n2.5\n2.5